capstone/src/main/webapp/lesson-loader.js

```diff
@@ -184,4 +184,17 @@ function createFormDivElement(lesson) {
     doc.getElementById("modal-form-url").href = lesson.entity.propertyMap.url;
     doc.getElementById("modal-form-url").innerHTML = "Click Here";
     return doc.body;
+}
+
+function createImageDivElement(lesson) {
+    let domparser = new DOMParser();
+    let doc = domparser.parseFromString(`
+        <div id="modal-image-description"></div>
+        <div class="small-spacing-bottom"></div>
+        <div id="modal-title"><span>Image</span></div>
+        <img class="img-responsive" id="image-container"></img>
+    `, "text/html");
+    doc.getElementById("modal-image-description").innerHTML = capitalizeFLetter(lesson.entity.propertyMap.description);
```

Comment:
innerHTML runs the DOM parser and allows XSS by adding a \<script> tag in the description. Use innerText